package.json

```diff
@@ -1,24 +1,23 @@
 {
   "name": "search-with-your-keyboard",
-  "version": "2.0.0",
+  "version": "0.0.0-development",
```

Comment:
Is this version change intended?

Reply:
We use that version in all other semantic-releases repos. He seems use some own stuff for calculating version.